src/factories/cf_apig.js

```diff
@@ -343,10 +346,7 @@ export function templateDeployment ({
       'Type': 'AWS::ApiGateway::Deployment',
       'Properties': {
         'RestApiId': { 'Ref': `${templateAPIID()}` },
-        'Description': `Automated deployment by danilo on ${date}`,
-        'StageName': 'dummy' // From the docs: "This property is required by API Gateway.
```

Comment:
Internal note: CFN was updated to not require a dummy StageName (previously required to create a Deployment when no stage did exist): https://aws.amazon.com/about-aws/whats-new/2016/10/aws-cloudformation-adds-improved-permissions-controls-cros-stack-reference-enhancements-and-resource-coverage-updates/